src/Diagnostics/FxCop/System.Runtime.Analyzers/VisualBasic/Usage/DisposableFieldsShouldBeDisposed.vb

```diff
@@ -29,19 +28,35 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.FxCopAnalyzers.Usage
                     Case SyntaxKind.SimpleMemberAccessExpression
                         ' NOTE: This cannot be optimized based on memberAccess.Name because a given method
                         ' may be an explicit interface implementation of IDisposable.Dispose.
+                        ' If the right hand side of the member access binds to IDisposable.Dispose
```

Comment:
Seems like lot of code in this method is very similar to the that in C# analyzer, except for just few language specific syntax types. We can probably refactor this to share more code, if you think that's worthwhile :)